Twig/NormalizingNodeVisitor.php

```diff
@@ -26,14 +26,14 @@
  *
  * @author Johannes M. Schmitt <schmittjoh@gmail.com>
  */
-class NormalizingNodeVisitor implements \Twig_NodeVisitorInterface
+class NormalizingNodeVisitor extends \Twig_BaseNodeVisitor
 {
     /**
      * @param \Twig_NodeInterface $node
      * @param \Twig_Environment $env
      * @return \Twig_NodeInterface
      */
-    public function enterNode(\Twig_NodeInterface $node, \Twig_Environment $env)
```

Comment:
Modify the container based on whether interface_exists ( '\Twig_NodeInterface' ) to use the v1/v2 classes?